### PR TITLE
Integrate tailwind css and create new button library

### PR DESCRIPTION
**Pull request by Pilopa**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Please ensure the "what" and "why" are explained properly in this description as it will be used as the commit description on squashing your pull request commits. 

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

**Contributor**:
- [ ] PR is well described and the description can be used as a commit message on squash
- [ ] Related issues linked to PR if existing and labels set
- [x] New/updated ui components are tested inside the storybook (module ui-components only) 

**Reviewer**:
- [ ] The clean code principles are respected ([CleanCode](https://clean-code-developer.com/virtues/))
- [ ] All new code/logic is implemented on the right spot / "Should this be done here?"
- [ ] UI changes fits into the layout
- [ ] The UI views / components are responsive (mobile views)
- [ ] Correct translations are available

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: scm-ui/ui-buttons/package.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-router-dom": "^5.3.1",
+    "classnames": "^2.2.6"
+  },
+  "devDependencies": {
+    "@scm-manager/ui-scripts": "2.37.3-SNAPSHOT",
+    "@scm-manager/ui-styles": "2.37.3-SNAPSHOT",
+    "@scm-manager/ui-api": "2.37.3-SNAPSHOT",
+    "@scm-manager/eslint-config": "^2.16.0-1658403197",
```
Comment (eheimbuch) [inline scm-ui/ui-buttons/package.json:32]: Bump here

File: scm-ui/ui-syntaxhighlighting/src/worker/SyntaxHighlighter.worker.ts [hunk 2]
```diff
@@ -91,7 +92,7 @@ const countChildrenAndApplyMarkers = (node: RefractorNode, markedTexts?: string[
   }
 
   if (newChildren.length > 0) {
-    const el = node as RefractorNode;
+    const el = node as unknown as RefractorElement;
```
Comment (eheimbuch) [inline scm-ui/ui-syntaxhighlighting/src/worker/SyntaxHighlighter.worker.ts:95]: :)

**Comment by eheimbuch** [comment]
Contrasts are quite poor for the tertiary buttons with dark backgrounds. I know it's not part of the story....

**Comment by eheimbuch** [comment]
I found some ui glitches:
Screenshot from 2022-08-01 09-53-05
Screenshot from 2022-08-01 09-52-48
Screenshot from 2022-08-01 09-52-16
Screenshot from 2022-08-01 09-51-35

File: scm-ui/ui-syntaxhighlighting/src/worker/SyntaxHighlighter.worker.ts (hunk 2 above)
Comment (Pilopa) [reply scm-ui/ui-syntaxhighlighting/src/worker/SyntaxHighlighter.worker.ts:95]: Its a necessary hack to get `tsc` happy, i know :'(

**Comment by Pilopa** [comment]
> I found some ui glitches: Screenshot from 2022-08-01 09-53-05 Screenshot from 2022-08-01 09-52-48 Screenshot from 2022-08-01 09-52-16 Screenshot from 2022-08-01 09-51-35

Damn I was pretty sure I fixed that. Thats due to the overwritten Bulma classes. I'll have a look.

**Comment by Pilopa** [comment]
> Contrasts are quite poor for the tertiary buttons with dark backgrounds. I know it's not part of the story....

Yeah, we would love to improve the contrast as well but it will have to be part of a follow-up story.

**Pull request merged**